app/graphql/types/invoices/object.rb

```diff
@@ -32,7 +32,9 @@ class Object < Types::BaseObject
       field :created_at, GraphQL::Types::ISO8601DateTime, null: false
       field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
 
+      # NOTE: will be removed when front-end will implement invoice versioning
```

Comment:
Well, we do not use this `legacy` attribute in our code so you can probably remove it already

Reply:
What is it used for on your side ? I assume it's for calculation logic, but wondering if it's either a issue on old invoices on UI side, or if it does not have any UI impact at all

Reply:
@ansmonjol 
It's used on old invoice where coupons, and prepaid credit were applied before VAT and to prevent creation of credit notes on this invoices.
It probably means there is an issue with this on the front-end, but since these invoice are now quite old and probably attached to test accounts, I think it's not a big deal...